### ISSUE TITLE
Updated Troubleshooting card headers

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2806]
Troubleshooting cards on the troubleshooting pages need a more distinct header. Duplicating the H2 as an H5 on the page violates Section 508 for distinct headers. This exists on most Troubleshooting pages

Example: [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdlVwIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--fc212e7c50311a9446687582e96445462daab638/image.png)

**Comment by jpetersononrr** [comment]
I was denied access to the zenhub file.

**Comment by cthomasONRR** [comment]
@jpetersononrr You might have to log out and back in... it's just a screen grab. I can go over this issue with you when you have time today.
